Core/src/main/java/com/plotsquared/core/plot/Plot.java

```diff
@@ -554,6 +555,16 @@ public void deleteMeta(final @NonNull String key) {
         return players;
     }
 
+    /**
+     * @param executor The player for whom the players in the plot should be retrieved
+     * @return list of visible PlotPlayer(s) for the executor inside of {@code this} plot
+     * @see #getPlayersInPlot()
+     * @since TODO
+     */
+    public @NonNull List<PlotPlayer<?>> getPlayersInPlotVisible(@NonNull PlotPlayer<?> executor) {
```

Comment:
Not sure if a public method in `Plot` is worth it. I'd prefer to have a (private) method in the command as long as it's only used there.